packages/cactus-plugin-ledger-connector-ubiquity/src/main/typescript/ubiquity.ts

```diff
@@ -0,0 +1,260 @@
+import type { Express } from "express";
+import type { Server as SecureServer } from "https";
+import type { Server } from "http";
+import {
+  Logger,
+  Checks,
+  LogLevelDesc,
+  LoggerProvider,
+} from "@hyperledger/cactus-common";
+import { PrometheusExporter } from "./prometheus-exporter/prometheus-exporter";
+import { v4 as uuidV4 } from "uuid";
+import { AxiosResponse } from "axios";
+import fetch from "node-fetch";
+import dotenv from "dotenv";
+import path from "path";
+
+import * as nodeFetch from "node-fetch";
+import { Optional } from "typescript-optional";
+import OAS from "../json/openapi.json";
+
+import { NETWORK_TYPE, PROTOCOL_TYPE } from "./helpers/api-parameters";
+import {
+  ConsensusAlgorithmFamily,
+  ICactusPlugin,
+  IPluginLedgerConnector,
+  IPluginWebService,
+  IWebServiceEndpoint,
+} from "@hyperledger/cactus-core-api";
+import { GetTransactionsByAddressEndpoint } from "./endpoints/transaction/get-transactions-endpoint";
+import {
+  UbiquityClient,
+  TxPage,
+  Balance,
+  Tx,
+} from "@ubiquity/ubiquity-ts-client-modified";
+
+const envPath = path.join(__dirname, "../.env");
+dotenv.config({ path: envPath });
```

Comment:
@RafaelAPB Executing code outside of class methods is highly discouraged because it can cause crashes at import-time that are very hard to debug when it happens on the CI or in production.
Please move this code to either the constructor or the `onPluginInit` async handler depending on what is the best place for it (I think the constructor is fine because the documentation of the methods called do not say that it would be async)